roofit/roofitcore/inc/RooAbsCollection.h

```diff
@@ -51,8 +51,10 @@ class RooAbsCollection : public TObject, public RooPrintable {
   // of the original variables.
   RooAbsCollection(const RooAbsCollection& other, const char *name="");
   RooAbsCollection& operator=(const RooAbsCollection& other);
-  RooAbsCollection& assignValueOnly(const RooAbsCollection& other, Bool_t oneSafe=kFALSE) ;
-  void assignFast(const RooAbsCollection& other, Bool_t setValDirty=kTRUE) ;
+
+  void assign(const RooAbsCollection& other);
+  void assignValueOnly(const RooAbsCollection& other, bool oneSafe=false) ;
```

Comment:
Maybe rename `oneSafe` to what it actually does. I always found that a bad name.